mart/attack/composer.py

```diff
@@ -38,43 +63,55 @@ def __call__(
         else:
             raise NotImplementedError
 
-    @abc.abstractmethod
     def compose(
```

Comment:
Should this just be private now?

Reply:
Yes. Fixed in b58b1077ccd4b6036420f5aab37409003b0f1906